packages/odyssey-react/package.json

```diff
@@ -13,6 +13,7 @@
   },
   "dependencies": {
     "@okta/odyssey-react-theme": "^0.9.3",
+    "@react-aria/focus": "3.5.0",
```

Comment:
Pinned at 3.5.0, given the bundle size of 3.5.2 (and later?) is twice the size.  See https://bundlephobia.com/package/@react-aria/focus@3.5.0